template.yaml

```diff
@@ -0,0 +1,343 @@
+---
+parameters:
+- name: IMAGE
+  value: "quay.io/lightspeed-core/lightspeed-stack"
+  description: "Container image for the lightspeed-stack application"
+- name: IMAGE_TAG
+  value: latest
+  description: "Tag of the container image to deploy"
+- name: MCP_SERVER_URL
+  value: "http://assisted-service-mcp:8000/sse"
+  description: "URL for the Model Context Protocol (MCP) server that provides assisted installer functionality"
+- name: ROUTE_HOST
+  value: "assisted-chat.example.com"
+  description: "Hostname for the OpenShift route to access the chat interface"
+- name: REPLICAS_COUNT
+  value: "1"
+  description: "Number of pod replicas to deploy for high availability"
+- name: SERVICE_PORT
+  value: "8090"
+  description: "Port number on which the lightspeed-stack service listens"
+- name: STORAGE_PVC_NAME
+  value: "assisted-chat-data"
+  description: "Name of the persistent volume claim for storing application data"
+- name: STORAGE_SIZE
+  value: "10Gi"
+  description: "Size of the persistent volume for storing feedback, transcripts, and SQLite databases"
+- name: STORAGE_MOUNT_PATH
+  value: "/tmp/data"
+  description: "Container path where the persistent volume will be mounted"
```

Comment:
Is this temporary or does it really need to be persisted between restarts of the pods? If it is temporary, can we use an `emptyDir` instead of a PVC? 

If we really need to use a PVC, then the access mode of the PVC below would need to be `ReadWriteMany`, and we will need to ensure that multiple pods know how to coordinate access to that storage.

I'd very much prefer if we use an `emptyDir`, as then these concerns go away and the deployment is simpler.